app/permissions/decidim/action_delegator/permissions.rb

```diff
@@ -21,6 +23,36 @@ def permissions
 
       private
 
+      def allowed_delegation_action?
+        # Check that the required question verifications are fulfilled
+        return unless authorized?(:vote, delegation.grantee)
+
+        case permission_action.action
+        when :vote_delegation
+          toggle_allow(question.can_be_voted_by?(delegation.grantee) || delegation_granted_to?(delegation.grantee))
+        when :unvote_delegation
+          toggle_allow(question.can_be_unvoted_by?(user))
```

Comment:
we should replace this with `delegation.grantee` for consistency. That's because `user == delegation.grantee`.